Expand 90.1 Unitary Air Conditioners data

AmmarDeh opened the pull request:
Expand and update 90.1 Air Conditioning data

AmmarDeh committed:
Expand 90.1 Air Conditioning data

Expand and update 90.1 Air Conditioning data

AmmarDeh committed:
Minor changes to the AC table

Checked errata data and removed 90.1-2007 AirCooled Small Duct High Velocity entry accordingly.
Added more annotations.

lymereJ commented on building_energy_standards_data/database_tables/hvac_minimum_requirement_unitary_air_conditioners.py:
```diff
@@ -7,102 +7,110 @@
 maximum_capacity: NUMERIC
 start_date: TEXT
 end_date: TEXT
 minimum_seasonal_efficiency: NUMERIC
+minimum_seasonal_efficiency_2: NUMERIC
```
I think we should probably correct this. SEER stands for Seasonal Energy Efficiency Ratio, not seasonal efficiency.

lymereJ reviewed (changes requested):
The JSON file is too large so I have to put my comments here:
- The `end_date` for the `Replacements` PTAC may be incorrect.
- Small detail but _The terms under "application" previously called "New Construction"and "Replacement"._ should be _The terms under "application" previously called "New Construction" and "Replacements"._
- I think that for the newer code versions, the most recent requirements some of the air-cooled units are IEER only (see email that I sent)

AmmarDeh committed:
Corrections to AC table

Removed EER requirements and kept IEER only for some of the Air Conditioners in Table 6.8.1-1 in 90.1-2013, 90.1-2019 and 90.1-2022.
Changed "minimum_seasonal_efficiency" column name to "minimum_seasonal_energy_efficiency_ratio".

AmmarDeh committed:
Added a new column in the 90.1 AC table

Added a new column "minimum_coefficient_of_performance_no_fan_cooling" to capture efficiency requirements in 90.1-2019 and 90.1-2022 PRM.
